### DISCUSSION TITLE
Welcome to paperman Discussions!

### DISCUSSION THREAD

**Discussion by sjg20** [sjg20/paperman#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by newclique** [comment]
I think I know the answer to this already :( but your application does not support opening the old .max files from Visioneer's Paperport software back in the early 2000s does it? .max just happens to be the name of the extension you've chosen for your own application's output if I am understanding your documentation correctly. Is that right?

**Reply by sjg20** [reply]
It does actually support that old format, both for read and write. I cannot promise that writing does exactly the same thing as Paperport though, as I stopped using it soon after I got this software running.

**Reply by newclique** [reply]
Oh, wow. I will try to get it running, then, because my latest attempts with opening the files with actual PaperPort v11 (which sees the .max files and tried to open them) fails. Could be the files are corrupt in some way but I doubt it since there are so many. If nothing else I might be able to debug using your code and see what is happening. Thank you!

**Reply by sjg20** [reply]
What Linux distro are you using?

**Reply by newclique** [reply]
Ubuntu 22.

**Reply by sjg20** [reply]
OK, jammy should work, from here: https://github.com/sjg20/paperman/releases/tag/v1.3.1

The bleeding-edge version has quite a few new features though, so might not be as stable.

**Reply by newclique** [reply]
Outstanding. Thanks again.
